packages/web-twig/src/Resources/components/Grid/README.md

```diff
@@ -168,6 +168,34 @@ component itself using the [`cols` prop](#api) and might not need to set columns
 listing with 3 columns is easier to set using `cols="3"` on the `Grid` component than setting
 `columnStart` and `columnEnd` on each `GridItem`.
 
+### Grid Placement Props: `columnStart`, `columnEnd`, `rowStart`, `rowEnd`
+
+These props control the placement of grid items within a CSS Grid layout by specifying where the item starts and ends in rows and columns:
+
+- `columnStart`: Specifies the vertical grid line where the item begins.
+- `columnEnd`: Specifies the vertical grid line where the item ends. The item spans up to, but does not include, this grid line. For example, if columnEnd is "3", the item ends at the line before the third column.
+- `rowStart`: Specifies the horizontal grid line where the item begins.
+- `rowEnd`: Specifies the horizontal grid line where the item ends. The item spans up to, but does not include, this grid line. For example, if rowEnd is "4", the item ends at the line before the fourth row.
+
+#### Grid Placement Example
+
+```twig
+<GridItem
+  columnStart="2"
+  columnEnd="5" <!-- or 'span 3' -->
```

Comment:
Nitpicking – Because the value is not set with single quotes, but you would like to distinguish the value via formatting, maybe you could be more descriptive:

```suggestion
      columnEnd="5" <!-- or columnEnd="span 3" -->
```

?